airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/job.py

```diff
@@ -206,23 +220,26 @@ def _log_job_state_with_count(self) -> None:
             self._log_job_msg_count += 1
         else:
             message = f"Elapsed time: {self._job_elapsed_time_in_state} sec"
+            if self._job_last_rec_count > 0:
+                count_message = f". Lines collected: {self._job_last_rec_count}"
+                message = message + count_message
             self._log_state(message)
             self._log_job_msg_count = 0
 
     def _log_state(self, message: Optional[str] = None) -> None:
-        pattern = f"Stream: `{self.stream_name}`, the BULK Job: `{self._job_id}` is {self._job_state}"
+        pattern = f"Stream: `{self.http_client._name}`, the BULK Job: `{self._job_id}` is {self._job_state}"
```

Comment:
A minor complaint, but I think we should exercise rigor and try to avoid referencing CDK component's private fields. In this case the HttpClient's private '_name' field. We do this in a few places

It's a bit tedious to add back the stream_name as a init param, and this case is somewhat trivial, but I think we should avoid this habit. Exposing the http client's name is something we probably should have done before and we can do now, but if you don't want to block on another CDK fix

Reply:
It's very handy to have this inside of http_client, since we don't need to provide more `dummy` attrs to the instances of something like `job_manager`, but just pass the http_client with all the information pre-defined. 

So you recommend not to use this awesome feature, but what are the risks here you see?

Reply:
Updated the cdk to `3.9.3`, which includes the `name` property, and changed the references. Works as expected. Thanks!